core/src/main/java/org/bitcoinj/quorums/SimplifiedQuorumList.java

```diff
@@ -492,11 +492,13 @@ void checkCommitment(FinalCommitment commitment, StoredBlock prevBlock, Simplifi
             }
 
             log.info("Quorum: {}", commitment.quorumHash);
-            StringBuilder builder = new StringBuilder();
-            for (Masternode mn : members) {
-                builder.append("\n ").append(mn.getProTxHash());
+            if (Context.get().isDebugMode()) {
+                StringBuilder builder = new StringBuilder();
+                for (Masternode mn : members) {
+                    builder.append("\n ").append(mn.getProTxHash());
+                }
+                log.info(builder.toString());
             }
```

Comment:
On mainnet and testnet, this was printing a lot of information to the logs something like 1000 lines.

In Dash Wallet, debug mode is off, though perhaps for debug builds it should be on.